### DISCUSSION TITLE
Can't get timelining module to work

### DISCUSSION THREAD

**Discussion by cipherz404** [EricZimmerman/KapeFiles#574] [Q&A] [answered]
I am trying to find the timelining folder directly from github:  https://github.com/EricZimmerman/KapeFiles/tree/master/Modules but when I go there, I don't see it. In following [this blog post by Mari DeGrazia](http://az4n6.blogspot.com/2019/08/triage-collection-and-timeline.html), the instructions say " _The timeline modules I created should be in the \Modules\timelining subfolder. If you do not have this folder after syncing (Eric was working on implement the syncing of module subfolders at the time of this blog post) you will need to grab the timelining folder directly from github:_ 
https://github.com/EricZimmerman/KapeFiles/tree/master/Modules." 

Did something change? I can't build a mini-timeline as I am getting errors. I have uploaded the log
[2021-11-11T145616_ConsoleLog.txt](https://github.com/EricZimmerman/KapeFiles/files/7521511/2021-11-11T145616_ConsoleLog.txt)
.

**Comment by AndrewRathbun** [comment, marked as answer]
That blog post is from August 2019. A LOT has changed with KAPE since then. The Modules went through a massive reorganization recently (https://github.com/EricZimmerman/KapeFiles/pull/557) and continue to be refined as needed. The `timelining` folder was removed as a result as well as all the Modules related to it due to multiple Issues being raised about it not working as intended (https://github.com/EricZimmerman/KapeFiles/issues/474, https://github.com/EricZimmerman/KapeFiles/issues/367, as well as many brought up internally and outside of GitHub). If you want the Modules still, go back in time in the repo and grab a copy of it and stuff it in your `!Local` folder but Eric or I aren't going to maintain it because we frankly don't use it. Ideally, the author of the Module would revive it so if you want that, feel free to reach out to whoever it is (generic advice for any Target/Module if you're not going to do it yourself). https://github.com/mdegrazia/KAPE_Tools is where you could raise an Issue, if you wanted. 

Hopefully that helps. Let me now if you need anything else.

EDIT: Here's a link to a point in time when the Timelining Modules still existed: https://github.com/EricZimmerman/KapeFiles/tree/2177f5695a83e78cd0fefc6b033638f2cd5dda8d

**Comment by EricZimmerman** [comment]
we have no issues adding things back to KAPE either, but the modules must be revamped (or more to the point, the programs doing the work in the modules) so make them work in a more consistent and less error prone manner. This would need to be done by the module author
